CHANGELOG-developer.next.asciidoc

```diff
@@ -52,6 +52,7 @@ The list below covers the major changes between 7.0.0-rc2 and master only.
 - Replace `ACKCount`, `ACKEvents`, and `ACKLastEvent` callbacks with `ACKHandler` and interface in `beat.ClientConfig`. {pull}19632[19632]
 - Remove global ACK handler support via `SetACKHandler` from publisher pipeline. {pull}19632[19632]
 - Make implementing `Close` required for `reader.Reader` interfaces. {pull}20455[20455]
+- Remove `NumCPU` as clients should update the CPU count on the fly in case of config changes in a VM. {pull}23154[23154]
```

Comment:
Do we consider this a breaking change or bug fix?

Reply:
It's a publicly exported variable, so _technically_ a breaking fix.

Reply:
Although it is a change in reporting, most users will not notice. Users that will notice have had wrong data. Let's call it a fix. How about: `Fix CPU usage metrics on VMs with dynamic CPU assignment. ...`